opm/simulators/aquifers/BlackoilAquiferModel_impl.hpp

```diff
@@ -139,72 +139,32 @@ template <typename TypeTag>
 void
 BlackoilAquiferModel<TypeTag>::init()
 {
-    /*
-    const auto& comm = this->simulator_.vanguard().gridView().comm();
     const auto& aquifer = this->simulator_.vanguard().eclState().aquifer();
-    const auto& connections = aquifer.connections();
 
-
-    for (const auto& aq : aquifer.fetp())
-        aquifers_Fetkovich.push_back(AquiferFetkovich<TypeTag>(connections[aq.aquiferID], cartesian_to_compressed_, this->simulator_, aq));
-
-    for (const auto& aq : aquifer.ct())
-        aquifers_CarterTracy.push_back(AquiferCarterTracy<TypeTag>(connections[aq.aquiferID], cartesian_to_compressed_, this->simulator_, aq));
-    */
-
-    const auto& deck = this->simulator_.vanguard().deck();
-    const auto& comm = this->simulator_.vanguard().gridView().comm();
-
-    bool has;
-    if (comm.rank() == 0)
-        has = deck.hasKeyword("AQUCT");
-    comm.broadcast(&has, 1, 0);
-
-    if (has) {
-         if (comm.size() > 1)
-             throw std::runtime_error("Aquifers currently do not work in parallel.");
-
-        // updateConnectionIntensiveQuantities();
-        const auto& eclState = this->simulator_.vanguard().eclState();
+    if (aquifer.active()) {
```

Comment:
I usually prefer
```C++
if (! aquifer.active()) {
    return;
}
```
in these circumstances.  That saves a level of indentation.

Reply:
okay, some like early returns, some do not; changed.